### DISCUSSION TITLE
扩展 Shortcode - Mapbox | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#507] [General]
# 扩展 Shortcode - Mapbox | FixIt

mapbox shortcode 使用 Mapbox GL JS 库提供互动式地图的功能。

https://fixit.lruihao.cn/zh-cn/docs/content-management/shortcodes/extended/mapbox/

**Comment by Ryaang** [comment]
Mapbox如果因为信用卡等问题注册失败，可以参考此教程
https://www.bilibili.com/read/cv24878918/
